onnx/helper.py

```diff
@@ -865,10 +865,10 @@ def printable_graph(graph, prefix=''):  # type: (GraphProto, Text) -> Text
     graphs = []  # type: List[GraphProto]
     # body
     for node in graph.node:
-        pn, gs = printable_node(node, indent, subgraphs=True)
-        assert isinstance(gs, list)
-        content.append(pn)
-        graphs.extend(gs)
+        contents_subgraphs = printable_node(node, indent, subgraphs=True)
+        assert isinstance(contents_subgraphs[1], list)
+        content.append(contents_subgraphs[0])
+        graphs.extend(contents_subgraphs[1])
```

Comment:
L868 and L800 both bumped into the following error: `error: Unpacking a string is disallowed` so I used single variable to catch it.